arch/X86/X86MappingInsnOp_reduce.inc

```diff
@@ -5267,11 +5267,11 @@
 },
 {	/* X86_SHRD16mrCL, X86_INS_SHRD: shrd{w}	$dst, $src2, cl */
 	X86_EFLAGS_UNDEFINED_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ, CS_AC_READ, CS_AC_READ, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_READ, CS_AC_READ, 0 }
 },
 {	/* X86_SHRD16mri8, X86_INS_SHRD: shrd{w}	$dst, $src2, $src3 */
 	X86_EFLAGS_UNDEFINED_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ, CS_AC_READ, CS_AC_IGNORE, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_READ, CS_AC_READ, 0 }
```

Comment:
yes, you are right. that is why CS_AC_READ must be changed it back to CS_AC_IGNORE